src/metabase/query_processor/streaming/xlsx.clj

```diff
@@ -412,6 +412,15 @@
   formatting can be applied. This should be enabled for generation of pulse/dashboard subscription attachments."
   false)
 
+(defn- maybe-parse-temporal-value
+  [value col]
+  (when (and *parse-temporal-string-values*
+             (isa? (:effective_type col) :type/Temporal)
+             (string? value))
+    (try (u.date/parse value)
```

Comment:
I think that we have enough info available to be fairly confident and drop the `try` here.

Reply:
I'm a bit wary of that, because I could imagine a column incorrectly having a temporal type but actually being a non-parsable string value. Not sure if that's likely, but if an exception is thrown here it renders the entire export invalid, so I think it's safer to keep the `try`